src/components/Admin/DataGrid.js

```diff
@@ -319,14 +323,15 @@ export const DataGrid = createReactClass({
             }
         });
 
-        if(!invalidData){
+        if (!invalidData) {
             const mutatedRows = this.state.rows.filter(row=>modifiedRows.has(row[this.props.rowKey]));
             //only save the grid rows that were modified to minimize unneccesary service mutations.
             modifiedRows.clear();
-            this.setState({localAction: STATE_ACTIONS.SAVING, filters: {}, modifiedRows: modifiedRows});
+            const selectedRowKeys = [];
+            this.setState({localAction: STATE_ACTIONS.SAVING, filters: {}, modifiedRows, selectedRowKeys});
```

Comment:
Nit: Maybe this would be clearer by in-lining the empty list:

```js
this.setState({selectedRowKeys: []})
```

This makes it immediately clear that we're clearing out the list without having to check the variable value on another line.